geoportal/geoportailv3_geoportal/static-ngeo/js/Theme.js

```diff
@@ -14,7 +20,33 @@ import appModule from './module.js';
  * @param {app.Themes} appThemes The themes services.
  * @ngInject
  */
-const exports = function($window, ngeoLocation, appThemes) {
+const exports = function(
+  $window,
+  ngeoLocation,
+  appThemes,
+  appStateManager,
+  appScalesService,
+  maxExtent
+  ) {
+
+  /**
+   * @type {app.StateManager}
+   * @private
+   */
+  this.appStateManager_ = appStateManager;
+
+  /**
+   * @type {ol.Extent}
+   * @private
+   */
+  this.maxExtent_ =
+      transformExtent(maxExtent, 'EPSG:4326', 'EPSG:3857');
```

Comment:
Is this still needed (as the `maxExtent` [is handled by v4](https://github.com/Geoportail-Luxembourg/luxembourg-geoportail/pull/124/files#diff-c02ef72735395e36c85dcd3761894da9409629acfa7cd3e7e911b6ffa6acbdfdR38)) ?

Reply:
you are right, it is no longer needed. Thx for pointing this.